scrypto-test/src/environment/builder.rs

```diff
@@ -246,6 +246,7 @@ where
                 };
 
                 System {
+                    versioned_system_logic: VersionedSystemLogic::V1,
```

Comment:
I might understand this as I review, but I assume that in here it doesn't matter if we do system v1 or v2 since the test environment doesn't deal with instructions or transactions at all so having it just be set to v1 or v2 should not matter. Is there any other changes in the system logic that could be useful to the transaction environment that aren't just support for new transactions?